test/e2e-tests.sh

```diff
@@ -24,12 +24,17 @@ source $(dirname $0)/e2e-common.sh
 
 cd $(dirname $(readlink -f $0))/../
 
+E2E_SKIP_CLUSTER_CREATION=${E2E_SKIP_CLUSTER_CREATION:="false"}
+
 ci_run && {
-  header "Setting up environment"
-  initialize $@
+    header "Setting up environment"
+    if [ "${E2E_SKIP_CLUSTER_CREATION}" != "true" ]; then
+        initialize "$@"
+    fi
 }
 
 tkn() {
+    header "Preparing tkn binary"
```

Comment:
shouldn't this be `Executing tkn binary` ?

Reply:
I removed this line for now, i think i need to refactor that a bit